packages/oasis-actions/src/strategies/ajna/earn/depositAndAdjust.ts

```diff
@@ -53,7 +38,8 @@ export async function depositAndAdjust(
     .then((res: any) => res.toString())
     .then((res: string) => new BigNumber(res))
 
-  let data: string | null = null
+  let data = ''
+  let targetPosition: AjnaEarnPosition | null = null
```

Comment:
would it be easer for you if you would assign current position to target position?